pathweaver.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lengthUnit": "FOOT",
+  "exportUnit": "Always Meters",
+  "maxVelocity": 10.0,
+  "maxAcceleration": 60.0,
+  "trackWidth": 2.0,
+  "gameName": "Infinite Recharge"
```

Comment:
We are not using pathweaver, also we don't use meters nor are we playing infinite recharge.